index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <link rel="stylesheet" href="style.css" />
```

Comment:
```suggestion
      <link rel="stylesheet" href="style.css" />
      <meta charset="utf-8">
```
Щоб кирилиця відображалась нормально на більшості девайсів не забувайте про цей мета тег